From 0b4e50486f509a972c0fee8388e6b9fb0e96309b Mon Sep 17 00:00:00 2001
From: Rabeeh Khoury <rabeeh@solid-run.com>
Date: Tue, 8 Jul 2014 00:33:28 +0300
Subject: [PATCH] Load brcmfmac.ko on boot

On LK 3.0.35, loading and unloading the driver isn't stable; so just keep
it loaded all the way.

Signed-off-by: Rabeeh Khoury <rabeeh@solid-run.com>
---
 cuboxi/rc.wifi |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/cuboxi/rc.wifi b/cuboxi/rc.wifi
index 0ef0a9e..1159326 100755
--- a/cuboxi/rc.wifi
+++ b/cuboxi/rc.wifi
@@ -26,6 +26,8 @@
 
 #insmod /system/lib/modules/bcmdhd.ko firmware_path=/etc/firmware/brcm/brcmfmac-sdio.bin nvram_path=/etc/firmware/brcm/brcmfmac-sdio.txt
 insmod /system/lib/modules/brcmutil.ko
+insmod /system/lib/modules/brcmfmac.ko
+
 #insmod /system/lib/modules/brcmfmac.ko brcmf_msg_level=1
 # Following enables tracing, error and iscan
 #insmod /system/lib/modules/brcmfmac.ko brcmf_msg_level=8195
-- 
1.7.9.5

